### ISSUE TITLE
index error

### ISSUE THREAD

**Issue by mclearc** [SvenWerlen/moulinette-core#63]
This may well be user error but I keep seeing this issue: 

```
Moulinette FileUtil | Couldn't load source moulinette/images/custom/index.json
Moulinette FileUtil | Couldn't load source moulinette/tiles/custom/index.json
```

I have, however, indexed both of these and there are `index-mtte.json` files to prove it. 

I've kept running into this even with repeated clean installs. I'm running the latest stable foundry (10.291) and am trying to index the basic tier of baileywiki material.

**Comment by SvenWerlen** [comment]
If you have new versions, moulinette is not supposed to look for `index.json` but only for `index-mtte.json`. 
Can you provide more information about which versions do you use?

**Comment by mclearc** [comment]
I just downloaded it yesterday through foundry so I expect it is the most current but I can double check when I'm by my computer

**Comment by mclearc** [comment]
Yeah just to confirm:
<img width="960" alt="Screenshot 2023-05-22 at 21 38 20" src="https://github.com/SvenWerlen/moulinette-core/assets/1502422/338201f4-2e50-4b20-a380-f9591ce314be">

**Comment by mclearc** [comment]
Here's another screenshot of the error in case that's helpful.
<img width="1512" alt="Screenshot 2023-05-23 at 16 17 23" src="https://github.com/SvenWerlen/moulinette-core/assets/1502422/d2cebb57-21bd-4c26-970d-908b3e0630d6">

**Comment by SvenWerlen** [comment]
Good catch! 
Thanks to your screenshots, I've been able to find that it came from the Moulinette Favorites. 
I fixed the issue in `moulinette-tiles` `v-11.0.4`.
Feel free to re-open it if you think it's not yet fixed.

**Comment by mclearc** [comment]
Works perfectly -- thanks very much!
